packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts

```diff
@@ -116,6 +116,10 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * The items that the base picker should currently display as selected. If this is provided then the picker will act as a controlled component.
    */
   selectedItems?: T[];
+  /**
+   * A callback used to sanitize the input string.
+   */
+  onInputChanged?: (input: string) => string;
```

Comment:
There is already a "change" event; we are inconsistent on past tense vs present. I'd suggest going with present and calling this "onInputChange" unless there is a good reason. What do you thing?

Reply:
Fair. I was going off of the prop 2 above (onItemSelected) and used the same tense. I'm fine either way.